### PR TITLE
5 files Changed

### PR DESCRIPTION
**Pull request by Adel-Laali**

### CODE REVIEW

File: contents/path/design/helicase.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-﻿<h1>هلیکاز</h1>
+# Helicase
 
-<p>شبیه به</p>
+#### Definition:
+#### Helicases are enzymes that bind and may even remodel nucleic acid or nucleic acid-protein complexes.
```
Comment (Nefcanto) [inline contents/path/design/helicase.md:4]: No definition needs to be presented as a header. Please remove those 4 hash signs.

File: contents/path/design/histone.md [hunk 2]
```diff
@@ -1,11 +1,4 @@
...
-</p>
-<p>
-    حالا توی گوگل سرچ کنین histone، <a target="_blank" href="https://www.google.com/search?q=histone&rlz=1C1OKWM_enIR912IR912&sxsrf=ALeKk00zRXsJzTcxEp_RTB30VYEmSvi_zg:1600689074979&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiZ0ta9l_rrAhVBZMAKHehlBJYQ_AUoAXoECBYQAw&biw=1273&bih=918">یا روی این لینک کلیک کنین</a>. هیستون یه پروتئینه که توی هسته سلول قرار داره، و DNA که خیلی بلنده، دور هیستون پیچیده میشه تا فضای کمتری رو اشغال کنه و بتونه داخل کروموزوم جا شه.
-    حالا آیا میتونین یه قرقره رو وسط یه کوه پیدا کنین، و بعد بگین "این طراحی نشده، این شانسی اینجا ایجاد شده"؟
-    چطوره که ما پیچیده شدن یه چیز طولانی به دور یه چیز دیگه رو در دنیای خودمون طراحی به حساب میاریم. اما در دل طبیعت وقتی بهش برخورد میکنیم اون رو نتیجه تکامل (شانس + زمان + بقای مطابق ترین) میدونیم؟
-</p>
+# Histone
+
+### Definition:
+### Family of basic proteins that associate with DNA in the nucleus and help condense it into chromatin. Nuclear DNA does not appear in free linear strands; it is highly condensed and wrapped around histones in order to fit inside of the nucleus and take part in the formation of chromosomes.
```
Comment (Nefcanto) [inline contents/path/design/histone.md:4]: This is not *title-based* or outlined. It's a full explanation. We don't do that in the first step. Please remove the explanation. Let's just keep things limited to titles, lists and tabular comparisons. In this case, we can simply create such a table:

Chromatin => Thread
Histone => Spool

File: contents/path/design/hormone-and-regulation.md [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
-<p>به عنوان مهندس از شما درخواست میشه وضعیت یه سیستم رو پایدار نگه دارین.</p>
+### Regulation Examples:
+- Satellites
+- Audio & Video Resolution
+- Refrigerator Temperature 
+- Pipelines Pressure
 
-<p>راه حلی که به ذهنتون میرسه چیه؟</p>
-<p>پایش و اصلاح. چیزی که به اسم رگلاژ میشناسیم.</p>
+### Concise: Monitoring & Correction
```
Comment (Nefcanto) [inline contents/path/design/hormone-and-regulation.md:9]: I guess these have technical terms. But we'll improve files over and over. This is good for  start.

File: contents/path/design/sphincter.md [hunk 4]
```diff
@@ -1,10 +1,15 @@
-﻿<h1>اسفنکتر</h1>
+# Sphincter
+ 
+#### Definition:
+#### A ring-shaped muscle that relaxes or tightens to open or close a passage or opening in the body.
```
Comment (Nefcanto) [inline contents/path/design/sphincter.md:4]: Again, never use headings for explanations. Use simple text (paragraphs), or use **bolding** in markdown.

And there is no need to explain it in this phase. We only list keywords, list examples, and compare via tables.

**Review by Nefcanto — CHANGES_REQUESTED** [review]
Thank you @Adel-Laali for the time. Please improve a couple of items and we're good to go.

**Comment by Adel-Laali** [comment]
Hi saeid, I understood that we shouldn't use explanation. but in some cases, I couldn't use lists instead explanation. So I need your guidance to fix them.

File: contents/path/design/common-base-for-production.md [hunk 5]
```diff
@@ -1,21 +1,14 @@
...
-    برای اینکه ما معادلش رو داشته باشیم در دنیای مهندسی باید بتونیم این حرف رو بزنیم:
-</p>
-<ul>
-    <li>موبایل؟ از x</li>
-    <li>ماشین؟ از x</li>
-    <li>خونه؟ از x</li>
-    <li>لباسشویی؟‌از x</li>
-</ul>
\ No newline at end of file
+| Gene     	| Crude oil      	|
```
[the remark this answers is not in the document]
Comment (Adel-Laali) [reply contents/path/design/common-base-for-production.md:3]: I see. You mean comparing Gene and Crude oil in the same table, isn't right. So I compare those, in different lists. wouldn't you agree?

File: contents/path/design/cortical-granule.md [hunk 6]
```diff
@@ -1,21 +1,27 @@
...
+
+***
+
+Sperm and Eggs:
+
+23 Chromosomes (Haploid).
+
+ Sex cells:
+ 
+ 46 Chromosomes (Diploid). gather from daddy & mommy for Spermatogenesis.
```
[the remark this answers is not in the document]
Comment (Adel-Laali) [reply contents/path/design/cortical-granule.md:17]: yes :)))), I knew that, but I wanted to make it funnier. I'll fix that.

File: contents/path/design/helicase.md (hunk 1 above)
Comment (Adel-Laali) [reply contents/path/design/helicase.md:4]: I'll fix it.

File: contents/path/design/histone.md (hunk 2 above)
Comment (Adel-Laali) [reply contents/path/design/histone.md:4]: This case is too hard to make simple. Its definition was also too long, so I made it short. however, it's better that you fix it yourself.

Thanks for your guidance.

**Comment by Nefcanto** [comment]
@Adel-Laali  I gave you a sample example related to the histone file. Please read it. Thank you.

File: contents/path/design/hormone-and-regulation.md (hunk 3 above)
Comment (Adel-Laali) [reply contents/path/design/hormone-and-regulation.md:9]: I agree with you. But there is no way to use another phrase or words. At least I didn't find that.

File: contents/path/design/sphincter.md (hunk 4 above)
Comment (Adel-Laali) [reply contents/path/design/sphincter.md:4]: Yes, yes. I got that we don't use headings for the explanation. And we don't USE explanations. Again, I couldn't make it to the list. It's too complicated.
In these types of cases, I just make them more simple and translate them into English.
Other parts need teammate's thoughts.

Thanks for your time.

**Comment by Adel-Laali** [comment]
About cortical-granule, I couldn't make it more simple and translate.

File: contents/path/design/Heart-and-Engine.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Heart & Engine
```
Comment (Nefcanto) [inline contents/path/design/Heart-and-Engine.md]: @Adel-Laali , while the content is good, the file name change is not good. Our convention is to use lowercase and dashes. This convention for casing and naming is called kebab-casing. Please lowercase "H" in Heart and "E" in Engine.

**Review by Nefcanto — APPROVED** [review]
Thanks again for the rest of your changes. I would approve them in this case. But we need to improve them over and over.

**Pull request merged**